src/components/common/GlobalAlert.js

```diff
@@ -203,58 +204,56 @@ const GlobalAlertNew = ({ globalAlert, actionStatus, clearGlobalAlert, closeIcon
-                                </Header>
-                                <div className='message-code'>
-                                    <Translate>
-                                        {({ translate }) => 
-                                            (typeof translate(alert.messageCode) === 'string' ? translate(alert.messageCode) : '').includes('No default translation found!')
-                                                ? <Translate id={`reduxActions.default.${alert.success ? 'success' : 'error'}`} />
-                                                : <Translate id={alert.messageCode} data={alert.data} />
-                                        }
-                                    </Translate>
-                                    {!alert.success && alert.messageCode &&
-                                        <Translate>
-                                            {({ translate }) => {
-                                                const msgCode = typeof translate(alert.messageCode) === 'string' ? translate(alert.messageCode) : '';
-                                                return (
+                <Translate>
+                    {({ translate }) =>
+                        <>
+                            {alerts.filter(alert => alert.show).map((alert, i) => {
+                                const msgCode = typeof translate(alert.messageCode) === 'string' ? translate(alert.messageCode) : '';
+                                const noTranslationFound = msgCode.includes('No default translation found!');
+                                return (
+                                    <Alert key={`alert-${i}`} success={alert.success} closing={closing} className={`number-${alert.type}`} type={alert.type}>
+                                        <Content>
+                                            <Icon>
+                                                <img src={alert.success ? IconCheckCircleImage : IconsAlertCircleImage} alt=''/>
+                                            </Icon>
+                                            <Text>
+                                                <Header success={alert.success}>
+                                                    <Translate id={alert.messageCodeHeader || (alert.success ? 'success' : 'error')} />
+                                                </Header>
+                                                <div className='message-code'>
+                                                    {noTranslationFound
+                                                        ? <Translate id={`reduxActions.default.${alert.success ? 'success' : 'error'}`} />
+                                                        : <Translate id={alert.messageCode} data={alert.data} />
+                                                    }
+                                                    {!alert.success &&
```

Comment:
Yep since we want to always show the 'View FAQ' button if `!alert.success`. The Zendesk URL will have a `query` param equal to `alert.errorMessage`, `alert.messageCode`, or just blank (no query)